app/assets/javascripts/covid19/src/webapp/components/proteome/ProteomePath.tsx

```diff
@@ -31,7 +31,7 @@ export const ProteomePath: React.FC<ProteomePathProps> = React.memo(props => {
     }, [name, setProteomeSelected, setSearch, toggleProteome]);
 
     const showProteinInfo = React.useCallback(() => {
-        setTitle(<span>{name}</span>);
+        setTitle(name);
```

Comment:
This title now is fixed to "SARS-CoV-2 Proteome", right? What's this state for?

Reply:
The title status is for the details title, when user hovers the protein.  It was already being used.